hcipy/wavefront_sensing/pyramid.py

```diff
@@ -17,18 +18,43 @@ class ModulatedPyramidWavefrontSensorOptics(WavefrontSensorOptics):
 		The modulation radius in radians.
 	num_steps : int
 		The number of steps per modulation cycle.
+	fast_modulation_method : boolean
```

Comment:
You should explain what the (dis)advantages of the fast modulation method are.

Reply:
done

Reply:
It's missing the disadvantages still. There need to be some, otherwise we would always use the fast method.

And I'd put this text in the main docstring rather than inside the parameters section.

Eg.:

```
'''The optical elements for a modulated pyramid wavefront sensor.

This class supports both a slow (default) and a fast modulation method, chosen by the
fast_modulation_method parameters. While the fast modulation method can be up to
twice as fast, it ....

Parameters
--------------
```

Reply:
I think the only disadvantage is that it uses more memory. The two methods give really similar results so I don't know which one is actually more accurate compared to a true modulated pyramid. I went with:

`This class supports both a slow (default) and a fast modulation method, chosen by the
fast_modulation_method parameters. The fast method shifts the pyramid surface instead of
applying tilts to the incoming wavefront. While the fast modulation method can be up to
twice as fast, it needs to precompute all pyramid surfaces and will use up more memory.`

Reply:
That sounds good. I just wanted something as the disadvantage. Otherwise there is no reason to not remove the slow method altogether.